docs/intro.rst

```diff
@@ -164,8 +164,7 @@ for symbols of a model without using labels.
 >>> with model.lock():
 ...     for symbol in model.iter_decisions():
 ...         symbol.set_state(0, [2])
-...     model.objective.state(0) == -4
-True
+...     assert model.objective.state(0) == -4
```

Comment:
I'm not sure about this: it save one line but at the cost of slightly less understandable code for less experienced users and the replacement of a False output were it to not obtain by an error state, which is not desirable

Reply:
The problem is that `NumPy` either returns `True` or `np.True_` depending on the version. So doctests may fail in either case. I believe `assert` is fairly clear even to novice Python users.

Reply:
Or could print the value

Reply:
Is that really clearer than `assert`? Both are built-in Python operations. If we cannot assume that they know about `assert`, then can we assume they know `print`?

Reply:
Yes, they all know print

Reply:
Originally we moved from `print()` statements to using equals to get a True because we worried about changes to the format of the output over time (mostly unordered lists and sets) but here that's not a concern

Reply:
I am comfortable assuming that they know `assert` as well.

Reply:
> changes to the format of the output over time (mostly unordered lists and sets) but here that's not a concern
Is it not? `NumPy` could decide to change the `str()` representation of `np.True_`.

Reply:
Sorry, I meant print the value: `print(model.objective.state(0))` and showing its output (that will be used by the doctest)

Reply:
That will return `np.float64(4)` or `4` depending on the NumPy version